assets/css/index.css

```diff
@@ -21,17 +21,18 @@ main{
 /* ScrollBar */
 /* width */
 ::-webkit-scrollbar {
-  width: 10px;
+  width: 15px;
```

Comment:
Do not increase the width of the scrollbar

Reply:
@arpitghura ok updated